test/evergreen.yml

```diff
@@ -885,15 +885,16 @@ tasks:
       - func: "compile wiredtiger"
         vars:
           posix_configure_flags: -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/clang.cmake -DENABLE_STRICT=1 -DHAVE_DIAGNOSTIC=0
-      - func: "compile wiredtiger"
```

Comment:
This issue is tracked in WT-11239. Newer branches have dedicated `CLANG_C_VERSION` and `CLANG_CXX_VERSION` flags but they're missing in the `configure wiredtiger` task, effectively not being run.
WT-11239 will fix this but it's uncovered some platform issues with libclang that will also need fixing on 6.0.
The cleanest solution for now is to disable and backport WT-11239

Reply:
Sounds good, thank you. Should we prioritise [WT-11239](https://jira.mongodb.org/browse/WT-11239)?

Reply:
I think it's ok right now. I've got the ticket moving in the background and I don't think we're losing much coverage here. We're still testing on both gcc and clang which is the important part. Note as part of WT-11239 I'm also discussing if we should add tests for versions 9 through 15 which we currently don't cover.